en/docs/learn/design-api/publish-api/publish-an-api-with-artifact-synchronization.md

```diff
@@ -0,0 +1,158 @@
+#Publish an API with Artifact Synchronization
```

Comment:
Let's get this reviewed by a team member too. @pubudu538  @tharindu1st  can one of you please review this document?